mscore/musescore.cpp

```diff
@@ -8020,7 +8020,6 @@ void MuseScore::init(QStringList& argv)
                   QRectF psf = p.paperRect(QPrinter::Inch);
                   MScore::defaultStyle().set(Sid::pageWidth,  psf.width());
                   MScore::defaultStyle().set(Sid::pageHeight, psf.height());
-                  MScore::defaultStyle().set(Sid::pagePrintableWidth, psf.width()-20.0/INCH);
```

Comment:
does that mean my PR is not needed anymore? Why can this be removed? Wasn't it there for a purpose?

Reply:
You mean #7007 ?

Reply:
this code tries to set A4 sizes on top of the style defaults, only offsetting it by an odd amount. At the same time, A4 values are already in the style itself. So this code has no value, only harm

Reply:
then shouldn't the other values by deleted too?

Reply:
I think there is no point in all this "if", I just didn't have time to check it out